### DISCUSSION TITLE
supabase tables eror

### DISCUSSION THREAD

**Discussion by phi2505** [supabase/supabase#15410] [Questions]
so, im having problem when signing up. becouse for some strange reasone when passing data on auth sign up. Some of the data dossent get inserted in to the database. Here is my code        

 const { data, error } = await supabase.auth.signUp({
          email: email,
          password: password,
          options: {
            data: {
              user_text: personlia, // Combine all the answers into a single string,
              user_job_text: this.api,
              full_name: Name,
              soknader: "1",  
            }
          }
        },
        
 
    );

and here is my table
 
create table
  public.profiles (
    id uuid not null,
    soknader text null,
    full_name text null,
    user_text text null,
    jobLink text null,
    user_job_text text null,
    constraint profiles_pkey primary key (id),
    constraint profiles_id_fkey foreign key (id) references auth.users (id)
  ) tablespace pg_default;

the soknader is just given null. 

I whould be very grathfull if someone could just tip me in the right direction. im quite new to supabase, so could be something realy simple thats wrong, whats the most strange thing for me is that, the user is made and some of the colums get data. But thwn why not one of them.

**Comment by GaryAustin1** [comment]
You need to show your trigger function and how you are setting each of those columns from your json data.
